release/cloudbuild.yaml

```diff
@@ -71,25 +71,27 @@ steps:
       - go
       - run
       - github.com/transparency-dev/armored-witness/cmd/manifest
+      - create
       - -git_tag=${TAG_NAME}
```

Comment:
I think these flags need to be double hyphens.

Reply:
Thanks for the catch, done!